a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -1120,41 +1105,37 @@ <h4>Variables setup</h4>
                         <li><b>LET</b> <var>page_break_markers</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>pageBreakMarkers</i>"]</code>.</li>
                         
 						<li><b>LET</b> <var>ruby_annotations</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>rubyAnnotations</i>"]</code>.</li>
-    				</ol>
+                        
+                        <li><b>LET</b> <var>sign_language</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>signLanguage</i>"]</code>.</li>
 
+                       <li><b>LET</b> <var>tactile_graphic</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>tactileGraphic</i>"]</code>.</li>
+                        
+                        <li><b>LET</b> <var>tactile_object</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>tactileObject</i>"]</code>.</li>
+                        
+ 						<li><b>LET</b> <var>text_to_speech_hinting</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>ttsMarkup</i>"]</code>.</li>
+    				</ol>
+                        
 					<h4 id="additional-accessibility-information-instructions">Instructions</h4>
 					<ol class="condition">
+                        <li>
+                            <span><b>IF</b> <var>page_break_markers</var>:</span>
+                            <span><b>THEN</b> display <code id="additional-accessibility-information-page-breaks">"Page breaks"</code>.</span>
```

Comment:
I'm still concerned with this section that we need to consider some other way of outputting these statements if we expect them alphabetized. It's like we need to push them to an array and sort it prior to displaying the statements.

Reply:
Sounds like we are not as concerned with outputting the additional accessibility information in alphabetical order.
I am fine with whatever order we think is meaningful be that the items we expect to see always come first and more obscure items follow.  Any thoughts @gregoriopellegrino @GeorgeKerscher?